libtextsecure/message_receiver.js

```diff
@@ -650,7 +650,9 @@ MessageReceiver.prototype.extend({
         var address = libsignal.SignalProtocolAddress.fromString(from);
 
         var ourNumber = textsecure.storage.user.getNumber();
-        var number = address.toString().split('.')[0];
+        var split = address.toString().split('.');
```

Comment:
nit: I know this predates this change, but if `address` is an instance of `SignalProtocolAddress`, using the `getNumber()`/`getDevice()`  is more readable than getting into the habit of `toString().split('.')[0]`.